File: pi-lens/tests/fixtures/tool-smoke/jedi-alt/bad.py
# Alternate-primary fixture: the Jedi LSP (jedi-language-server) is the fallback
# for the Python file kind when the default `python` server (pyright) is
# disabled. The missing colon after the def is a syntax error that
# jedi-language-server publishes as a diagnostic on didOpen.
def greet(name)
    print(name)

File: permission&sandbox/pi-lens/tests/fixtures/tool-smoke/deno-alt/bad.ts
// Alternate-primary fixture: the Deno LSP (deno lsp) is the fallback for the
// TypeScript file kind when the default `typescript` server is disabled. The
// sibling deno.json both satisfies DenoServer's root detector (deno.json /
// deno.jsonc, no fallback) and puts the workspace in Deno mode so the server
// type-checks. The type error below yields a TS2322 diagnostic from deno-ts.
const greeting: number = "not a number";
console.log(greeting);
